lib/pluginsNgsi2/addEvent.js

```diff
@@ -0,0 +1,37 @@
+
+/*
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-iotagent-lib
+ *
+ * fiware-iotagent-lib is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * fiware-iotagent-lib is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with fiware-iotagent-lib.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::daniel.moranjimenez@telefonica.com
+ *
+ * Modified work Copyright 2017 Atos Spain S.A
```

Comment:
this is not the way to state your authorship. You can add a sentence like

`Author: Daniel Calvo Atos Spain S.A. ` 

IPR issues are currently being discussed by Fiware Foundation and current IPR owners

Reply:
An example (currently used at Orion repository) of the style proposed by @jmcanterafonseca:

https://github.com/telefonicaid/fiware-orion/blob/master/src/lib/serviceRoutinesV2/optionsGetOnly.cpp#L23

Reply:
In the case of an existing file, "Modified by:" would be better.